google/cloud/storage/internal/grpc/stub_failures_test.cc

```diff
@@ -45,6 +45,11 @@ class GrpcClientFailuresTest
         client_(
             storage_experimental::DefaultGrpcClient(TestOptions(GetParam()))) {}
 
+  void SetUp() override {
+    // Older versions of gRPC flake on this test, see #13114.
+    if (grpc::Version() < "1.60.0") GTEST_SKIP();
```

Comment:
In practice, seems fine. Would skip "1.100.0" and not skip "1.9.0" for example.

Reply:
Fixed.